docs/howto/initial-data.txt

```diff
@@ -80,16 +80,32 @@ from the fixture and reloaded into the database. Note this means that if you
 change one of the rows created by a fixture and then run :djadmin:`loaddata`
 again, you'll wipe out any changes you've made.
 
-Where Django finds fixture files
---------------------------------
+Telling Django where to look for fixture files
```

Comment:
I would suggest: _How to tell Django..._ or maybe _Tell Django..._ (verbs are good for how-to guides).

Reply:
I like the second option more, we have "How to" in the title of the whole guide.